src/shared/components/MetadataCard/index.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import * as moment from 'moment';
+import { Card, Button, Tooltip, Divider, Descriptions } from 'antd';
+import { Resource } from '@bbp/nexus-sdk';
+
+import TypesIcon from '../Types/TypesIcon';
+import Copy from '../Copy';
+import { getUsername, getResourceLabel } from '../../utils';
+
+import './MetadataCard.less';
+
+const MetadataCardComponent: React.FunctionComponent<{
```

Comment:
My issue with this component is its name... `MetadataCard` of what? You basically don't know until you look at the code. Can we rename to something like `ResourceCard` or `ResourceMetadataCard` ?

Reply:
I like ResourceCard :)